src/antsibull_changelog/update_galaxy.py

```diff
@@ -0,0 +1,35 @@
+# Author: Sagar Paul <paul.sagar@yahoo.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2023, Ansible Project
+
+"""
+Update version in galaxy.yml.
+"""
+
+from __future__ import annotations
+
+from .yaml import load_yaml, store_yaml
+
+
+class UpdateGalaxy:
```

Comment:
Would this be better off as a function? Having `__init__` set two attributes and then call a method with side effects seems strange to me.

Reply:
Yes, @gotmax23 That makes way more sense, I thought of adding it that way but we need not create a separate class just for it. Updated the PR as per it.